docs/en/admins/04_Updating.md

```diff
@@ -0,0 +1,31 @@
+# Backing Up and Updating FreshRSS
+
+The following is general procedure; [specific commands for Linux may be found here](07_LinuxUpdate.md).
+
+## Backing Up
```

Comment:
We should add the option to import/export by SQLite. There is another documentation issue left open for that